### ISSUE TITLE
Incorrect naming

### ISSUE THREAD

**Issue by BuildItBusk** [dotnet/EntityFramework.Docs#2454]
The private backing field is named "_url" but the configuration refers to "_validateUrl". 

![image](https://user-images.githubusercontent.com/39243399/84503910-57325680-acbb-11ea-89a2-596beffb48fe.png)

![image](https://user-images.githubusercontent.com/39243399/84503953-6c0eea00-acbb-11ea-9cd7-93fbbdabbb9c.png)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 2e162696-b19f-d73f-8b1f-d1b4c3048b62
* Version Independent ID: a83e4fb5-e4ae-a46f-7248-487ea2679ac5
* Content: [Backing Fields - EF Core](https://docs.microsoft.com/en-us/ef/core/modeling/backing-field?tabs=fluent-api)
* Content Source: [entity-framework/core/modeling/backing-field.md](https://github.com/dotnet/EntityFramework.Docs/blob/master/entity-framework/core/modeling/backing-field.md)
* Product: **entity-framework**
* Technology: **entity-framework-core**
* GitHub Login: @rowanmiller
* Microsoft Alias: **avickers**

**Comment by ajcvickers** [comment]
@BuildItBusk The example here is showing how to configure the backing field if it doesn't match the property name. It is not intended to refer to the exact same code sample as above. This seems clear from the DataAnnotation tab, but could be clearer in the fluent API tab.

**Comment by BuildItBusk** [comment]
> @BuildItBusk The example here is showing how to configure the backing field if it doesn't match the property name. It is not intended to refer to the exact same code sample as above. This seems clear from the DataAnnotation tab, but could be clearer in the fluent API tab.

Fair enough. I just assumed that the backing field would match the property name. Either way, it's not a huge problem, as the key concept of the example is still very understandable :-)

**Comment by ajcvickers** [comment]
@BuildItBusk The point is that the backing field does not need to be specified if it the naming aligns with the property name. This example is showing how to specify the backing field when the field and the property are differently named.
